src/main/resources/reference.conf

```diff
@@ -3,4 +3,5 @@ email {
   password = ""
   port = 587
   host = "smtp.gmail.com"
+  timeout = 10000
```

Comment:
does it have to be called reference.conf? can't it be something else? either in application.conf or make some explicit email.conf files (at work we would keep all such things in application.conf). But this is purely up to you to do this change

Reply:
reference.conf file is mainly there to show what it should look like - real config should be manually put in application.conf - i figured not pushing application.conf to git would prevent possible conflicts with local config when more things are added to reference.conf